docs/docs/how-tos/domain-registry.md

```diff
@@ -78,7 +78,10 @@ The amount of time this takes varies for each DNS provider. Validate such inform
 
 Qhub has an extra flag for deployments that grants management and the creation of the DNS records for you automatically. For automatic DNS provision add `--dns-auto-provision` to your qhub deploy command:
 
-`qhub deploy -c qhub-config --dns-provider cloudflare --dns-auto-provision`
+```console
+qhub deploy -c qhub-config.yaml --dns-provider cloudflare --dns-auto-provision
```

Comment:
```suggestion
qhub deploy -c qhub-config.yaml \
  --dns-provider cloudflare \
  --dns-auto-provision
```

making it easier to read in small screens

Reply:
Thank you!